tabs/mission_control.js

```diff
@@ -307,6 +310,8 @@ TABS.mission_control.initialize = function (callback) {
                 $('#pointLon').val(coord[1]);
```

Comment:
The Latitude and Longitude are transposed see #364 and #366.

Lines 306 and 307 corrected:
```
$('#pointLon').val(coord[0]);
$('#pointLat').val(coord[1]);
```